composer.json

```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
+        "ext-mongo": "mongodb extetension to use MongoDB writer",
         "zendframework/zend-escaper": "Zend\\Escaper component, for use in the XML formatter",
-        "zendframework/zend-mail": "Zend\\Mail component",
-        "zendframework/zend-validator": "Zend\\Validator component"
+        "zendframework/zend-mail": "Zend\\Mail component to use Email writer",
+        "zendframework/zend-validator": "Zend\\Validator component to use filter feature"
```

Comment:
Hi! Can you [see this page](https://github.com/zendframework/zend-log/search?utf8=%E2%9C%93&q=Validator)..

Zend\Validator is used to use filtering feature IMO